packages/paste-core/components/card/src/index.tsx

```diff
@@ -4,34 +4,29 @@ import {PaddingProps, isPaddingTokenProp} from '@twilio-paste/style-props';
 
 export interface CardProps extends React.HTMLAttributes<HTMLElement>, PaddingProps {}
 
-const Card: React.FunctionComponent<CardProps> = ({
-  children,
-  padding = 'space60',
-  paddingBottom,
-  paddingLeft,
-  paddingRight,
-  paddingTop,
-  ...props
-}) => {
-  return (
-    <Box
-      {...safelySpreadBoxProps(props)}
-      as="article"
-      borderWidth="borderWidth20"
-      borderColor="colorBorderLighter"
-      borderStyle="solid"
-      borderRadius="borderRadius20"
-      padding={padding}
-      paddingBottom={paddingBottom}
-      paddingLeft={paddingLeft}
-      paddingRight={paddingRight}
-      paddingTop={paddingTop}
-      backgroundColor="colorBackgroundBody"
-    >
-      {children}
-    </Box>
-  );
-};
+const Card = React.forwardRef<HTMLElement, CardProps>(
```

Comment:
fast coming to the conclusion we should basically make this a requirement for all components to forward refs

Reply:
Yeah. I'll make a ticket to do a forwardRef pass. It's pretty important